bin/prep-release.php

```diff
@@ -56,6 +56,7 @@ function get_files() {
         "src/libmongoc/src/libbson/VERSION*",
         "src/libmongoc/src/libmongoc/src/mongoc/*.{c,def,defs,h,h.in}",
         "src/libmongoc/src/zlib-1.*/*.{c,h,h.in}",
+        "src/libmongoc/src/utf8proc-2.*/*.{c,h,h.in}",
```

Comment:
```suggestion
        "src/libmongoc/src/utf8proc-2.*/*.{c,h}",
```

There are no [configuration files](https://www.gnu.org/software/autoconf/manual/autoconf-2.71/html_node/Configuration-Files.html) in utf8proc (unlike zlib), so we can remove this.